### PR TITLE
HOTFIX Add excel import to default pages

### PR DESCRIPTION
**Pull request by Kanol**

### CODE REVIEW

File: apps/condo/domains/common/components/EmptyListView.tsx [hunk 1]
```diff
@@ -62,13 +63,24 @@ export const EmptyListView: React.FC<IEmptyListProps> = ({ label, message, creat
...
                 {message}
             </Typography.Text>
-            <Button
-                type={'sberDefaultGradient'}
-                style={{ marginTop: '24px' }}
-                onClick={() => router.push(createRoute)}
-            >
-                {createLabel}
-            </Button>
+            <Row gutter={[10, 0]} align={'middle'} justify={'center'} style={{ marginTop: '24px' }}>
```
Comment (gegius) [inline apps/condo/domains/common/components/EmptyListView.tsx:66]: You should move all styles to constants (align and justify too)

It is also recommended to take out the gutter

const _ : RowProps['gutter']

File: apps/condo/domains/common/components/EmptyListView.tsx [hunk 2]
```diff
@@ -52,7 +53,7 @@ export const BasicEmptyListView: React.FC<IBasicEmptyListProps> = ({
     )
 }
 
-export const EmptyListView: React.FC<IEmptyListProps> = ({ label, message, createRoute, createLabel }) => {
+export const EmptyListView: React.FC<IEmptyListProps> = ({ label, message, button, createRoute, createLabel }) => {
     const router = useRouter()
     return (
```
Comment (gegius) [inline apps/condo/domains/common/components/EmptyListView.tsx:58]: Blank line before return statement

File: apps/condo/domains/common/components/EmptyListView.tsx (hunk 1 above)
Comment (AleX83Xpert) [inline apps/condo/domains/common/components/EmptyListView.tsx:66]:
```suggestion
            <Row gutter={[10, 0]} align="middle" justify="center" style={{ marginTop: '24px' }}>
```
Also, it's good to move `gutter` and `style` definitions ti constants.

File: apps/condo/domains/common/components/EmptyListView.tsx [hunk 3]
```diff
@@ -62,13 +63,24 @@ export const EmptyListView: React.FC<IEmptyListProps> = ({ label, message, creat
...
-                type={'sberDefaultGradient'}
-                style={{ marginTop: '24px' }}
-                onClick={() => router.push(createRoute)}
-            >
-                {createLabel}
-            </Button>
+            <Row gutter={[10, 0]} align={'middle'} justify={'center'} style={{ marginTop: '24px' }}>
+                {
+                    button ?
+                        (
```
Comment (AleX83Xpert) [inline apps/condo/domains/common/components/EmptyListView.tsx:69]:
```suggestion
                    button ? (
```

File: apps/condo/domains/common/components/EmptyListView.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { Empty, EmptyProps, Space, Typography } from 'antd'
+import { Empty, EmptyProps, Space, Typography, Row, Col } from 'antd'
 import { useRouter } from 'next/router'
 import { Button } from './Button'
 import { EmptyIcon } from './EmptyIcon'
```
Comment (gegius) [inline apps/condo/domains/common/components/EmptyListView.tsx:4]: It should to write full path

File: apps/condo/domains/property/components/BuildingsTable.tsx [hunk 5]
```diff
@@ -116,6 +116,22 @@ export default function BuildingsTable (props: BuildingTableProps) {
         ? <EmptyListView
             label={EmptyListLabel}
             message={EmptyListMessage}
+            button={(<ImportWrapper
```
Comment (gegius) [inline apps/condo/domains/property/components/BuildingsTable.tsx:119]:
```suggestion
            button={(
            <ImportWrapper
```

File: apps/condo/domains/property/components/BuildingsTable.tsx [hunk 6]
```diff
@@ -116,6 +116,22 @@ export default function BuildingsTable (props: BuildingTableProps) {
...
+                objectCreator={propertyCreator}
+            >
+                <Button
+                    type={'sberPrimary'}
+                    icon={<DiffOutlined/>}
+                    secondary
+                />
+            </ImportWrapper>)}
             createRoute="/property/create"
             createLabel={CreateProperty}/>
```
Comment (AleX83Xpert) [inline apps/condo/domains/property/components/BuildingsTable.tsx:136]: What about improving a code style a bit?
```suggestion
        ? (
        <EmptyListView
            label={EmptyListLabel}
            message={EmptyListMessage}
            button={(
                <ImportWrapper
                    objectsName={PropertiesMessage}
                    accessCheck={role?.canManageProperties}
                    onFinish={refetch}
                    columns={columns}
                    rowNormalizer={propertyNormalizer}
                    rowValidator={propertyValidator}
                    domainTranslate={PropertyTitle}
                    objectCreator={propertyCreator}
                >
                    <Button
                        type={'sberPrimary'}
                        icon={<DiffOutlined/>}
                        secondary
                    />
                </ImportWrapper>
            )}
            createRoute="/property/create"
            createLabel={CreateProperty}
        />
        )
```

File: apps/condo/pages/meter/index.tsx [hunk 7]
```diff
@@ -117,6 +117,31 @@ export const MetersPageContent = ({
...
+                                    button={(
+                                        <ImportWrapper
+                                            objectsName={MeterReadingImportObjectsName}
+                                            accessCheck={canManageMeterReadings}
+                                            onFinish={refetch}
+                                            columns={columns}
+                                            maxTableLength={hasFeature('bigger_limit_for_import') ?
+                                                EXTENDED_RECORDS_LIMIT_FOR_IMPORT :
+                                                DEFAULT_RECORDS_LIMIT_FOR_IMPORT
+                                            }
```
Comment (AleX83Xpert) [inline apps/condo/pages/meter/index.tsx:129]: 😿 
```suggestion
                                            maxTableLength={
                                                hasFeature('bigger_limit_for_import')
                                                    ? EXTENDED_RECORDS_LIMIT_FOR_IMPORT
                                                    : DEFAULT_RECORDS_LIMIT_FOR_IMPORT
                                            }
```

**Review by AleX83Xpert — APPROVED** [review]
It will be nice to keep code style by pressing cmd+alt+L

File: apps/condo/domains/common/components/EmptyListView.tsx (hunk 4 above)
Comment (Kanol) [reply apps/condo/domains/common/components/EmptyListView.tsx:4]: > It should to write full path

Antd is full path for this elements

Comment (gegius) [reply apps/condo/domains/common/components/EmptyListView.tsx:4]: hah, no, I mean Button and Icon

**Review by gegius — APPROVED** [review]

**Pull request merged**